src/js/view/controls/components/button.js

```diff
@@ -1,6 +1,6 @@
 import UI from 'utils/ui';
 
-export default function (icon, apiAction, ariaText) {
+export default function (icon, apiAction, ariaText, svgIcons) {
```

Comment:
Let's revisit this later, about cleaning up the constructor to make it more readble and possibly letting `svgIcons` being and array or object:

const icons = isArray(svgIcons) ? svgIcons : [svgIcons];

That way we don't have to wrap a single icon arguments when called